packages/components/src/organisms/SKUMatrix/SKUMatrixSidebar.tsx

```diff
@@ -0,0 +1,300 @@
+    setIsOpen(false)
+  }
+
+  function handleAddToCart(e: React.MouseEvent<HTMLButtonElement, MouseEvent>) {
+    buyButtonOnClick(e)
+    onClose()
+  }
+
+  const totalColumnsSkeletonLength =
+    Object.keys(columns).filter((v) => v !== 'additionalColumns').length +
+    (columns.additionalColumns?.length ?? 0)
+
+  return (
+    <SlideOver
+      data-fs-sku-matrix-sidebar
+      size={size}
+      direction={direction}
+      overlayProps={overlayProps}
+      isOpen={isOpen}
+      fade={fade}
+      {...otherProps}
+    >
+      <SlideOverHeader onClose={onClose}>
+        <h2 data-fs-sku-matrix-sidebar-title>{title}</h2>
+      </SlideOverHeader>
+
+      {children}
+
+      <Table variant="bordered">
+        <TableHead>
+          <TableRow>
+            <TableCell align="left" variant="header" scope="col">
+              {columns.name}
+            </TableCell>
+
+            {columns.additionalColumns?.map(({ label, value }) => (
+              <TableCell key={value} align="left" variant="header" scope="col">
+                {label}
+              </TableCell>
+            ))}
+
+            <TableCell align="left" variant="header" scope="col">
+              {columns.availability.label}
+            </TableCell>
+
+            <TableCell align="right" variant="header" scope="col">
+              {columns.price}
+            </TableCell>
+
+            <TableCell align="left" variant="header" scope="col">
+              {columns.quantitySelector}
+            </TableCell>
+          </TableRow>
+        </TableHead>
+
+        <TableBody>
+          {loading ? (
+            <>
+              {Array.from({ length: 5 }).map((_, index) => {
+                return (
+                  <TableRow key={`table-row-${index}`}>
+                    {Array.from({
+                      length: totalColumnsSkeletonLength,
+                    }).map((_, index) => {
+                      return (
+                        <TableCell key={`table-cell-${index}`}>
+                          <span>
+                            <Skeleton
+                              key={index}
+                              size={{ width: '100%', height: '30px' }}
+                            />
+                          </span>
+                        </TableCell>
+                      )
+                    })}
+                  </TableRow>
+                )
+              })}
+            </>
+          ) : (
+            <>
+              {allVariantProducts.map((variantProduct) => (
+                <TableRow key={`${variantProduct.name}-${variantProduct.id}`}>
+                  <TableCell data-fs-sku-matrix-sidebar-cell-image align="left">
+                    <div>
+                      <Image
+                        src={variantProduct.image.url}
+                        alt={variantProduct.image.alternateName}
+                        width={48}
+                        height={48}
+                      />
+                    </div>
```

Comment:
```suggestion
                     <Image
                       src={variantProduct.image.url}
                       alt={variantProduct.image.alternateName}
                       width={48}
                       height={48}
                     />
```

nitpick: I don't think we need this <div> here 🤔